### DISCUSSION TITLE
37C3 community meet

### DISCUSSION THREAD

**Discussion by Erethon** [markqvist/Reticulum#386] [General]
With [37C3](https://events.ccc.de/congress/2023/infos/startpage.html) in Hamburg approaching fast (27-30/12) is there any interest for a meeting of the Reticulum community? If enough of us gather, we could try setting up a network in Hamburg over LoRa for the duration of the congress.

**Comment by markqvist** [comment]
Hey, that's a great initiative. I like the idea of this, and I'm actually going to be in that part of Europe at that time of the year, so I could probably be there as well.

**Comment by anm** [comment]
Yes, I am interested.

**Comment by Erethon** [comment]
The dates for the sale events are out https://events.ccc.de/en/2023/11/19/37c3-tickets-presale/.

I'll have some 868Mhz devices with me. If more people have boards that cover different bands we want to try setting up a mesh, let me know in advance so I can order boards that support other frequencies.

**Reply by anm** [reply]
I can bring a few 868MHz boards, but most are SX1262 which is apparently not supported. I have a LILYGO T-Beam v1.2 which I suspect is SX1276.

**Reply by Erethon** [reply]
As you noted, only SX1276 and SX1278 are [currently supported](https://github.com/markqvist/RNode_Firmware#supported-transceiver-modules) unfortunately. The T-Beam should indeed work however!

**Comment by Erethon** [comment]
As the deadline (3/12) for requesting an assembly approaches, do we think we have enough people/interest in setting one up? I would be happy to spent a couple of hours per day geeking about Reticulum and spreading the word, but I think we'll need 1-2 more people to help run this.

**Reply by jooray** [reply]
I'm interested. Both in meet and to play with the network

**Reply by anm** [reply]
Is there an assembly? I am at congress now anyway, with a beard I can put RNode on.

I am also organising the Maritime Hackers meetup, where there will be at least two talks about LoRa projects. It should also be possible to join remotely - details to be confirmed.

https://events.ccc.de/congress/2023/hub/en/event/maritime-hackers-meetup/

**Reply by Erethon** [reply]
> Is there an assembly?

I didn't setup an assembly as interest didn't seem that high and I wasn't sure of my availability during congress.

**Comment by jooray** [comment]
We have a network here.

My LXMF address: 03941c092abfa8574c2bf93bf20800a8

Sideband rnode settings:

RGQWNTRTWBWABILCZX2CJILUCGQXGCFBMMDKC2OAUFSMBILOUCQXBIA=

For some reason, it did not import well when I tried it, so screenshot
![image](https://github.com/markqvist/Reticulum/assets/1028688/b814dbef-daf8-4847-b781-71bb9e0b3e7a)

**Reply by Erethon** [reply]
BW of 62.5 seems a bit low, I think 250k might be better.

I'll have a phone with me running Sideband starting tomorrow and a t-beam. Unfortunately, the way my hotel room is structured I don't think I can easily run a rnode from the hotel room.

**Reply by jooray** [reply]
250 it is

Current config

RGQWNTRTWBWABILCZYAAHUEQUF2BDILTBCQWGBVBNHAKCZGAUFXKBILQUA======

Awaiting your first message

**Comment by Erethon** [comment]
We ended up meeting with @jooray and chatting over Reticulum during the day. We discussed about meeting again tomorrow around 14:30, let's say near the colorful unicorn by the coffee nerds/house of tea. We can discuss LoRa/Reticulum then.

**Reply by strafplanet** [reply]
There may be a few of us meshtastic people dropping by, too :-)

(See #37C3LoRa:strafpla.net / https://matrix.to/#/%2337C3LoRa%3Astrafpla.net)

**Reply by Erethon** [reply]
We had a small gathering of seven people and discussed about LoRa/Reticulum/Meshtastic a bit. Hopefully more people are onboarded to the network now ;)

I'll leave this discussion open until the end of 37c3 in case people want to coordinate more. There's is also [this meetup tomorrow](https://events.ccc.de/congress/2023/hub/en/event/maritime-hackers-meetup/) that's LoRa adjacent.

**Comment by Erethon** [comment]
37C3 is now over, so I'm closing this discussion.
